packages/remark-mark/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "remark-mark",
+  "version": "1.0.0",
+  "repository": {
+    "url": "https://github.com/zestedesavoir/zmarkdown/tree/master/packages/remark-mark",
+    "type": "git"
+  },
+  "author": "Victor Felder <victor@draft.li> (https://draft.li)",
```

Comment:
I think you can add yourself to this one :)

Reply:
Thanks :)